src/Moryx.Model.Sqlite/SqliteModelConfigurator.cs

```diff
@@ -67,8 +68,10 @@ public override DbContextOptions BuildDbContextOptions(IDatabaseConfig config)
         }
 
         private static string BuildConnectionString(IDatabaseConfig config)
-        {
-                return config.ConnectionSettings.ConnectionString;
+        {  
+           return  config.ConnectionSettings.ConnectionString.Contains("Pooling") ? 
```

Comment:
I'd prefer to use the connection string builder to set this up, instead of concatenating strings.